internal/handlers/commonparams/extract_params.go

```diff
@@ -29,18 +29,42 @@ import (
 	"github.com/FerretDB/FerretDB/internal/util/lazyerrors"
 )
 
+// ErrFieldNotPopulated is returned when a required field is not populated.
+var ErrFieldNotPopulated = errors.New("required field is not populated")
+
 // ExtractParams fill passed value structure with parameters from the document.
 // If the passed value is not a pointer to the structure it panics.
 // Parameters are extracted by the field name or by the `ferretdb` tag.
 //
 // Possible tags:
```

Comment:
What I don't see in this list is numbers that will be rounded to X if too big value is set. As far as I remember, this is how `scale` in `collStats`/`dbStats` work. But I guess we don't move `scale` yet, right?

Reply:
Yes, we move only `delete`, `distinct`, `find`, `findAndModify`, `insert` and `update`.